packages/account/src/Components/account-limits/account-limits.tsx

```diff
@@ -131,7 +133,7 @@ const AccountLimits = observer(
                             <AccountLimitsArticle is_from_derivgo={is_from_derivgo} />
                         )}
                         <div className='da-account-limits__table-wrapper'>
-                            <ThemedScrollbars is_bypassed={should_bypass_scrollbars || isMobile()}>
+                            <ThemedScrollbars is_bypassed={should_bypass_scrollbars ?? isMobile()}>
```

Comment:
are we doing such sonarcloud warnings for ourselfs? I don’t understand what is the reason of using here '??' Instead of '||".
I believe exactly here we need '||' operator, as 'should_bypass_scrollbars' is boolean value